ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsService.scala

```diff
@@ -308,8 +339,14 @@ class ContractsService(
       parties: OneAnd[Set, domain.Party],
       templateIds: Set[domain.TemplateId.RequiredPkg],
       queryParams: InMemoryQuery,
+  )(implicit
+      lc: LoggingContextOf[CorrelationID with RequestID]
   ): Source[Error \/ domain.ActiveContract[LfValue], NotUsed] = {
 
+    logger.debug(
+      s"Searching in memory, parties: $parties, templateIds: $templateIds, queryParms: $queryParams"
+    )
```

Comment:
Should parties and template identifiers be part of the logging context, rather than logged in the message? Putting the query parameters could also be reasonable but a bit challenging, because the logging context right now doesn't allow to nest objects.

Reply:
Well this one was kind of a suggestion what could be logged additionally.